proposals/meshery.md

```diff
@@ -1,85 +1,126 @@
+-   Exporting load test results to a common accepted Service Mesh Performance Specification (SMPS)
+
+The direction of the project has generally been guided by our open source community and users. There are a plethora of GitHub issues requesting various features that we prioritize based on popularity of user requests and engineering capacity.
+
+A roadmap for future features, including those listed above, can be found in GitHub at <https://github.com/layer5io/meshery>.
 
-A roadmap for future features, including those listed above, can be found in GitHub at https://github.com/meshery/meshery/ROADMAP.md
 The project welcomes contributions of any kind: code, documentation, bug reporting via issues, and project management to help track and prioritize workstreams.
 
-== Use Cases
-The following is a list of common use-cases for Meshery users:  
-
- * Performance management of service meshes and applications running on top of them.
- * Lifecycle management of the most popular service meshes.
- * 
-
-== CNCF Donation Details
- * *Preferred Maturity Level:* Sandbox
- * *Sponsors:* @leecalcote, 
- * *License:* Apache v2
- * *Source control repositories / issue tracker:* https://github.com/layer5io/meshery.
- * *Infrastructure Required:* N/A
- * *Website:* https://meshery.io/
- * *Release Methodology and Mechanics:* Documented at https://docs.meshery.io/releases
-
-== Social Media Accounts:
-
- * *Twitter:* https://twitter.com/mesheryio
- * *Youtube:* https://www.youtube.com/channel/UCFL1af7_wdnhHXL1InzaMvA/playlists
-  * *Slack:* http://slack.layer5.io
-
-== Contributor Statistics (valid as of Feb 2020)
- * 150 GitHub Stars
- * 1,605 Commits
- * 322 PRs
- * 625 Issues Opened
- * 44 Contributors
- * 40 releases
-
-== Asks from CNCF
- * A vendor-neutral home for Meshery to facilitate growth and community involvement
- * Logistics – General access to resource and staff to provide advice, and help optimize our growth
- * Infrastructure for CI / CD
- * Integration with CNCF devstat
-
-== Appendix
-
-=== Architecture
-Meshery is architected as a client of the Kubernetes API, leveraging Envoy. You can learn more about its architecture at https://docs.meshery.io/architecture
-
-== Landscape
-There are numerous service meshes controllers available for developers and platform architecture teams to leverage. An analysis of the various options is presented on https://layer5.io/landscape
\ No newline at end of file
+Use Cases 
+----------
+
+The following is a list of common use-cases for Meshery users:
+
+-   Service Meshes installation with default configuration
+
+-   Trying out sample applications on the service meshes
+
+-   Ability to run performance tests on applications
+
+-   Persist and compare results of different performance tests
+
+CNCF Donation Details
+---------------------
+
+-   Preferred Maturity Level: Sandbox
+
+-   Sponsors: @leecalcote, @justincormack, @michellenoorali,
```

Comment:
Can we add the links to their GitHub/Social Profiles ?

Reply:
Sounds good. FYI - convention in the CNCF is to use links to Twitter handles.

Reply:
Added Twitter handles with links